CONTRIBUTING.md

```diff
@@ -112,13 +112,15 @@ Testing Lints
 **Creating Unit Tests.** Every lint should also have corresponding unit
 tests (generally at least one for a success and one for a failure condition). We
 have typically generated test certificates using Go (see
-[documentation][CreateCertificates] for details), but OpenSSL
-could also be used. Test certificates should be placed in `testdata/` and called
-from the test file created by `newLint.sh`. You may want to prepend the PEM with
-the output of `openssl x509 -text`. You can run your lint against a test
-certificate from a unit test using the `test.TestLint` helper function.
+[documentation][CreateCertificates] for details and [example][certGenerator] as
+reference), but OpenSSL could also be used. Test certificates should be placed in
+`testdata/` and called from the test file created by `newLint.sh`. You may want to
+prepend the PEM with the output of `openssl x509 -text`. You can run your lint
+against a test certificate from a unit test using the `test.TestLint` helper
```

Comment:
@RufusJWB While you're editing this part of the docs would you mind updating this part:

> You may want to the output of `openssl x509 -text`. You can run your lint against a test	prepend the PEM with the output of `openssl x509 -text`. 

This is actually mandatory now (CI will fail otherwise) and the best way to do it is by running `test/prepend_testcerts_openssl.sh`. Maybe we can say something like:

> All test certificates **must** have the textual description from `openssl x509 -text` added before the PEM header or CI will flag them as a build error. You can add the text decoding to all of the test certs missing it by running `test/prepend_testcerts_openssl.sh`.

Reply:
done